src/main/java/org/fcrepo/migration/Migrator.java

```diff
@@ -37,27 +38,56 @@ public static void main(String [] args) throws IOException, XMLStreamException {
             Migrator m = new Migrator(new FoxmlDirectoryObjectSource(foxmlRoot, dsRoot, working),
                     new ConsoleLoggingStreamingFedoraObjectHandler());
             m.run();
-        }
+        }*/
+        final ApplicationContext context = new ClassPathXmlApplicationContext("spring/migration-bean.xml");
+        final Migrator m = (Migrator)context.getBean("migrator");
```

Comment:
Don't get beans by name and cast them. Use a properly type-safe generic method:

http://docs.spring.io/spring/docs/4.1.5.RELEASE/javadoc-api/org/springframework/beans/factory/BeanFactory.html#getBean-java.lang.String-java.lang.Class-